### PR TITLE
Pull define(...) method into a dedicated interface

### PR DESCRIPTION
**Pull request by jblievremont**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BaseQProfileWsAction.java [hunk 1]
```diff
@@ -19,13 +19,14 @@
...
 
+import org.sonar.api.server.ws.Definable;
 import org.sonar.api.server.ws.RequestHandler;
 import org.sonar.api.server.ws.WebService;
 
 /**
  * Marker interface for quality profile related web service end points
  */
-public interface BaseQProfileWsAction extends RequestHandler {
+public interface BaseQProfileWsAction extends RequestHandler, Definable<WebService.NewController> {
```
Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BaseQProfileWsAction.java]: why not create a WsAction interface which extends RequestHandler and Definable<WebService.NewController> ? then, when creating a new marker interface action, we would extend WsAction only. WDYT ?

Comment (jblievremont) [inline server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BaseQProfileWsAction.java]: :+1: on my side, but keeping WsAction as an internal interface of sonar-server (with RequestHandler and Definable<T> in sonar-plugin-api)

Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BaseQProfileWsAction.java]: :+1:

Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BaseQProfileWsAction.java]: Seems good to me

Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BaseQProfileWsAction.java]: @jblievremont Will you add the WsAction ?

Comment (jblievremont) [inline server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BaseQProfileWsAction.java]: Yep.

**Pull request closed without merging**